ros_cross_compile/pipeline_stages.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Wrapper abstractions for each pipeline stage.
+
+The abstractions allows for streamlining the pipeline into a unified
+'pipeline runner' and collecting of time series data from each stage.
+"""
+
+from pathlib import Path
+from typing import List, NamedTuple, Optional
+
+
+ConfigOptions = NamedTuple('ConfigOptions', [('skip_rosdep_collection', Optional[bool]),
```

Comment:
bools don't have to be optional, they can default to False. Optional(bool) is effectively a ternary value.